hack/observability/jaeger/fetch-jaeger-resources.sh

```diff
@@ -22,6 +22,7 @@ CHART_RELEASE=${CHART_RELEASE:-0.1.5}
 JAEGER_ROOT=$(dirname "${BASH_SOURCE[0]}")
 CHART_ROOT=$JAEGER_ROOT/chart
 
-rm -rf "${CHART_ROOT:?}/"*
+rm -rf "$CHART_ROOT"
 wget -qO- https://github.com/hansehe/jaeger-all-in-one/raw/master/helm/charts/jaeger-all-in-one-"$CHART_RELEASE".tgz \
-    | tar xvz -C "$JAEGER_ROOT" -s /^jaeger-all-in-one/chart/ -
+    | tar xvz -C "$JAEGER_ROOT" -
```

Comment:
I ran this on Linux and I'm getting this error:
```
tar: -: Not found in archive
tar: Exiting with failure status due to previous errors
```

It looks like my tar command defaults to:
``` 
*This* tar defaults to:
--format=gnu -f- -b20 --quoting-style=escape --rmt-command=/usr/lib/tar/rmt
```
Do you have a Linux setup by any chance to verify this error?

Reply:
Turns out that using dash (`-`) to represent the stdin file handle isn't actually supported by GNU `tar`. But apparently it's not necessary either, so removing it seems to make this work on both versions of `tar` in my testing.

Thanks @Jont828, LMK if it works for you!

Reply:
Seems like it's working on my end!